src/OrleansCounterControl/OrleansCounterControl.csproj

```diff
@@ -45,8 +45,24 @@
     <WarningsAsErrors>4014</WarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
I think we need to remove the "External Assembly References" section from http://dotnet.github.io/orleans/Contributing.